aries-site/src/pages/components/checkboxgroup.mdx

```diff
@@ -25,10 +25,11 @@ CheckBoxGroup should be used to present a group of options to the user. The user
 
 ### Best practices
 
-- CheckBoxGroup should be used to select zero or more of options from a list.
+- CheckBoxGroup should be used to select zero or more options from a list.
 - In a CheckBoxGroup, options are independent from one another.
 - Settings changed by checkboxes should not take effect until the user clicks to proceed.
-- It is recommended to have only 10 or fewer options in a CheckBoxGroup. Use multiple CheckboxGroups if necessary.
+- It is recommended to have only 4 or fewer options in a CheckBoxGroup.
+- Use [SelectMultiple](/components/selectmultiple#best-practices) with search when multiple selections can be made in a list that contains 10 or more items.
```

Comment:
Feels like we should also have the bullet here about using SelectMultiple when there are >4 but <10 items.

Reply:
```suggestion
- Use [SelectMultiple](/components/selectmultiple#best-practices) with search when multiple selections can be made in a list that contains 5-9 items; use [SelectMultiple](/components/selectmultiple#best-practices) with search when containing 10 or more items.
```